vnext/IntegrationTests/TestRunner.cpp

```diff
@@ -13,6 +13,8 @@
 #include <objbase.h>
 #include <locale>
 
+#include <boost/exception/all.hpp>
```

Comment:
Out of curiosity, what does this fix / enhance?

Reply:
I didn't see any real exception in testing yet, but I hope to provide more logs when exception happens.
At the beginning, I was trying the catch the assert which results in a crash.
Finally I found assert crash aborted the process and I can't catch it.
The blame would helps with the assert crash.